modules/ROOT/pages/insight.adoc

```diff
@@ -23,208 +35,269 @@ Insight helps you answer questions about your integrated apps, such as:
+.The screenshot shows (*1*) the Transactions level, (*2*), the Events level, and (*3*), the Metadata level on the Insight dashboard.
+image::rtm-insight-dashboard.png[Tracking levels on the Insight dashboard]
 
-[%header,cols="2*a"]
-|===
-|Action |To Apply
-|Display a specific transaction |
-Click *Insight*. Click *Advanced* in the Search field and in *Transaction ID*, type the ID value. Click *Apply*.
+Business data, if configured, appear in the Metadata area (*3*) of the dashboard. See <<Setting Up Business Events>>.
 
-image::chintransid.png[CHInTransID]
 
-|Display only transactions which failed |
-Indicate *Any*, *Completed*, or *Failed.*
+=== Search or Filter Transactions
 
-image::chinany.png[CHInAny]
+On the *Insight* page, you can search or filter data to refine the results displayed. You can search for:
 
-|Display transactions within a specific date range |
-* Click *Advanced* > *Date & Time* to specify a date or a range:
+* A specific transaction by transaction ID
+* Transactions within a specific date range
+* Transactions according to flow name, exception message, processing time, and/or any custom business data
++
+To enable the *Business Data* field your app must have business events configured. See <<Setting Up Business Events>>.
 
-image::chindatetime2.png[CHInDateTime2]
+To search or filter transactions:
 
-* Click and drag within the graph to select a specific date range
+. Click *Insight* in the menu on the left.
+. Enter your search criteria in the *Search* field.
+. Click *Advanced* in the *Search* field.
+. Specify filter options:
++
+.The screenshot shows the advanced search filter fields on the Insight dashboard.
+image::rtm-insight-search-filter.png[Advanced search filter fields on the Insight dashboard]
+// image::chintransid.png[CHInTransID]
++
+--
+* To display only failed or completed transactions, select *Failed* or *Completed* from the menu in the *Transaction ID* field.
++
+.The arrow shows the menu in the *Transaction ID* field.
+image::rtm-insight-filter-transactions.png[Advanced search filter fields on the Insight dashboard]
 
-image::ch-date-range-click-drag.png[CH_date+range+click+drag]
+* To display transactions within a specific date range, specify a date or range in the *Date & Time* fields.
++
+Alternatively, click and drag within the graph to select a specific date range:
++
+.The arrow shows the selected transaction range on the Insight dashboard.
+image::rtm-insight-filter-selected.png[Selected transaction range on the Insight dashboard]
+--
+. Click *Apply* on the *Advanced* pane to apply your search criteria.
 
-|Locate transactions according to flow name, exception message, processing time, and/or any custom business data |
-In your application, click *Advanced* in the *Search* field to locate additional search parameters.
+. If you want to save a search, click *Save search*:
```

Comment:
I don't think that the actions performed on a saved search are really subordinate tasks to saving a search: running, editing, and deleting.